examples/hybrid_jobs/1_Quantum_machine_learning_in_Amazon_Braket_Hybrid_Jobs/qcbm/qcbm_job.py

```diff
@@ -62,10 +69,19 @@ def callback(x):
         global iteration_number  # global
         iteration_number += 1
         loss = mmd_loss(qcbm.probabilities(x), data)
+
+        timestamp = time.time()
+
+        braket_cost = float(cost_tracker.simulator_tasks_cost() + cost_tracker.qpu_tasks_cost())
+
         log_metric(
-            metric_name="loss",
-            value=loss,
+            metric_name="braket_cost",
```

Comment:
Since the cost tracker is just an estimator of just Braket tasks, I think it would be good to rename braket_cost to something like braket_task_estimate or estimated_braket_task_cost to emphasize what the values represent and reduce opportunities for confusion.

Reply:
okay, updated to `braket_tasks_cost` to be consistent with the  `qpu_tasks_cost()` and `simulator_tasks_cost()` syntax